src/util.js

```diff
@@ -126,14 +126,14 @@ export async function domChange (fn) {
 }
 
 export function getHeadingLevel (h) {
-	if (h?.computedRole === "heading") {
+	if (h.computedRole === "heading") {
 		if (h.ariaLevel) {
 			return h.ariaLevel;
 		}
+	}
 
-		let tag = h.nodeName;
-		if (tag.length === 2 && tag[0] === "H" && tag[1] >= 1) {
-			return parseInt(tag[1]);
-		}
+	let tag = h.nodeName;
+	if (tag.length === 2 && tag[0] === "H" && tag[1] >= 1) {
+		return parseInt(tag[1]);
```

Comment:
Why? An `<h*>` element *also* has a computed role of "heading".

Reply:
I thought so, too. But when I set breakpoints to see why `openHeadings` doesn't calculate correctly, this code block wasn't rich a single time. 🤷‍♂️

Reply:
Here is what I see in the dev tools when inspecting an `<h3>`:

<img width="1258" alt="image" src="https://github.com/user-attachments/assets/b3da727f-a2a1-4792-ae4a-8a4730466351">